src/microceph.py

```diff
@@ -232,6 +232,30 @@ def join_cluster(token: str, micro_ip: str = "", **kwargs):
     _run_cmd(cmd=cmd)
 
 
+def enable_nfs(target: str, cluster_id: str) -> None:
```

Comment:
when enabled from charm we should always check the relation associated juju network space and use that binding as the bind_addr for enablement.